ledger/acctupdates.go

```diff
@@ -206,6 +215,9 @@ type accountUpdates struct {
 	// baseResources stores the most recently used resources, at exactly dbRound
 	baseResources lruResources
 
+	// baseKV stores the most recently used KV, at exactly dbRound
```

Comment:
I'd call it either `baseKVs` or `baseKVStore`, as it is a bunch of Key/Values. (right?)